deno/mod.ts

```diff
@@ -17,29 +27,39 @@ interface Hyper {
   };
 }
 
+const exists = (c: Count): Count =>
+  ifElse(
+    (c: Count): boolean => compose(is(Number), prop("count"))(c),
```

Comment:
If `count` is mistakenly cast to a non-number, ie the string `"5"`, then that count will be lost and reset to 0. 

Should this util simply check for existence of `count` and then attempt cast if it exists? Or is that out of this utils purview?

Reply:
It should always be a number

Reply:
I am going to leave it as implemented